README.md

```diff
@@ -103,9 +103,15 @@ you need a CoW-enabled filesystem on your `~/.cache/fingertip/machines`.
 In practice, this probably means either `btrfs` or specially-created `xfs`
 (see example below):
 
+When you first run `fingertip`, the setup wizard can create this for you.
+You can adjust this behavior with `FINGERTIP_SETUP=auto|suggest|never`
+and the created size using `FINGERTIP_SETUP_SIZE` (default is 25GB).
```

Comment:
'or interactively'. Or maybe even phase it somehow so that the average user won't try setting any of those on their first time at least. Something like 'it should cover the regular usage scenario just fine. In case you need to automate the wizard, you can set [no need to mention the default size] ... or perform the setup manually:'.

Reply:
I tried to reword it based on the above comment so I hope it is better :)
If you have a better wording in your mind, feel free to push it here instead of my poor attempt, delete it altogether or merge just the rest.

Reply:
It's OK now, I don't feel the urge to set those anymore before even trying to run the wizard.